qiskit/basics/getting_started.ipynb

```diff
@@ -48,47 +48,28 @@
    "outputs": [],
    "source": [
     "from qiskit import QuantumCircuit, ClassicalRegister, QuantumRegister\n",
-    "from qiskit import execute\n",
-    "from qiskit.tools.visualization import circuit_drawer\n",
-    "from qiskit.backends.ibmq import least_busy\n",
-    "from qiskit.wrapper.jupyter import *"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 2,
-   "metadata": {
-    "ExecuteTime": {
-     "end_time": "2018-09-29T00:04:18.108102Z",
-     "start_time": "2018-09-29T00:04:18.104498Z"
-    }
-   },
-   "outputs": [],
-   "source": [
-    "from qiskit import Aer"
+    "from qiskit import execute"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Basic Concept\n",
-    "\n",
-    "The basic concept of our Qiskit Terra is an array of quantum circuits. The workflow consists of three stages: [Build](#sectionB), [Compile](#sectionC), and [Run](#sectionR). Build allows you to make different quantum circuits that represent the problem you are solving; Compile allows you to rewrite them to run on different backends (simulators/real chips of different [quantum volumes](http://ibm.biz/qiskit-quantum-volume), sizes, fidelity, etc); and Run launches the jobs.  After the jobs have been run, the data is collected. There are methods for putting this data together, depending on the program. This either gives you the answer you wanted, or allows you to make a better program for the next instance.\n",
-    "\n",
-    "### Building your circuit: Create it  <a id='sectionB'></a>"
+    "The basic concept of Qiskit is an array of quantum circuits. The workflow consists of two stages: [Build](#sectionB), and [Execute](#sectionR). Build allows you to make different quantum circuits that represent the problem you are solving; Execute allows you to run them to run on different backends.  After the jobs have been run, the data is collected. There are methods for putting this data together, depending on the program. This either gives you the answer you wanted, or allows you to make a better program for the next instance."
```

Comment:
"Execute allows you to run them to run on different backends" should go to "Execute allows you to run them on different backends "